### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by pilhuhn**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (abonas) [inline .github/ISSUE_TEMPLATE/bug_report.md]: perhaps add "screenshots are helpful if you have them"

Comment (pilhuhn) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ about: Create a report to help us improve
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+For UI issues please also add a screenshot that shows the issue.
```
Comment (xeviknal) [inline .github/ISSUE_TEMPLATE/bug_report.md]: arent the UI issues going to be open into kiali/kiali-ui?

Comment (josejulio) [reply .github/ISSUE_TEMPLATE/bug_report.md]: kiali-ui issues are disabled, all issues are going here AFAIK

Comment (pilhuhn) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yes, only here. One place to remember is hard enough :-)
Also users that just get Kiali via docker container and want to report a bug will see this one.

**Comment by pilhuhn** [comment]
@josejulio Do you think it has to go into the template? It will certainly go into the contribution guidelines.

**Comment by josejulio** [comment]
> @josejulio Do you think it has to go into the template? It will certainly go into the contribution guidelines.

I think that is good to have some advice for someone who is filling a new issue, people might sometimes miss where to ask and will end up opening an issue for that.

Perhaps having a commented block at the beginning, linking to the contributing guidelines (or any other useful note) to avoid any question issue, see this [example](https://github.com/cherrypy/cherrypy/issues/new)

**Comment by pilhuhn** [comment]
Actually if they open a support question here, it means neither the description in the UI, docs nor the web site are good enough, so it could be seen as a legitimate bug report

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+ 
+**General remarks**
+
+> Please delete this section including header before submitting
```
Comment (josejulio) [inline .github/ISSUE_TEMPLATE/bug_report.md]: You can comment out this block, so the text won't be displayed once is submited 

```
<!---
Please delete etc
-->
```

But i'm fine to merge as it, is better than nothing :)

**Pull request merged**